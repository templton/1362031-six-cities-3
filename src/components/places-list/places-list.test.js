import React from "react";
import renderer from "react-test-renderer";
import PlacesList from "./places-list";

describe(`Render <PlacesList/>`, () => {
  it(`Empty render`, () => {
    const tree = renderer
      .create(<PlacesList places={[]} cardClass={`card_class`} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
