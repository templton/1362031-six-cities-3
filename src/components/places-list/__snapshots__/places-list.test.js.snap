// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <PlacesList/> Empty render 1`] = `null`;
